test/integration/query-interface/describeTable.test.js

```diff
@@ -128,14 +132,14 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
           expect(isAdmin.defaultValue).to.be.null;
       }
 
-      if (dialect.startsWith('postgres')) {
+      if (dialect.startsWith('postgres') || dialect.startsWith('yugabytedb')) {
```

Comment:
same comment about using `===` because you don't have to take `postgres-native` into account